source/MapDetailPanel.cpp

```diff
@@ -800,11 +800,16 @@ void MapDetailPanel::DrawInfo()
 			else
 			{
 				value -= localValue;
-				price += "(";
+				if(Preferences::Has(Preferences::PRICE_PARENS))
+					price += "(";
 				if(value > 0)
 					price += '+';
+				else if(value < 0)
+					price += '-';
+				value = abs(value);
```

Comment:
Wouldn't be the same without this code?

Reply:
At one point, I did not realize the difficulty of changing out the '-' character for a better fitting one.